index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Project 1: Set up and Mobile Version Skeleton</title>
+
+  <link rel="stylesheet" type="text/css" href="styles.css">
+
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="nav-list">
+        <li><p id="nav-bar-p">Welcome</p></li>
+        <li><button id="nav-bar-btn"><img src="images/Icon - Menu.png" alt="A hamburguer icon" width="35px" height="35px"></button></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="heading-section">
+    <h1 id="main-heading">Hey There. I'm Ulises <br><strong>I'm a Software Developer</strong></h1>
+
+    <p> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+    <div id="icons-div">
+      <img src="images/twitter.png" alt="twitter icon">
+      <img src="images/Linkedin icon.png" alt="linkedin icon">
+      <img src="images/medium.png" alt="A big M icon">
+      <img src="images/github.png" alt="github icon">
+      <img src="images/angellist.png" alt="two tumbs up icon">
+    </div>
```

Comment:
Please use `<ul>` and `<li>` tags here for semantics reasons, so that browsers can identify it as a list item